.gitignore

```diff
@@ -14,6 +14,7 @@ src/main/resources/docs/
 /preferences.json
 /*.log.*
 hs_err_pid[0-9]*.log
+/addressbookdata/
```

Comment:
thanks for catching this small change! yepp, addressbookdata should not be pushed :)